### PR TITLE
Fix strax updates into straxen and fix issues

### PR DESCRIPTION
**Pull request by jorana**
**What is the problem / what does the code in this PR do**
8dffccb  Fix deprecation issues due to https://github.com/AxFoundation/strax/pull/318
a70b630 Remove S3 from straxen https://github.com/AxFoundation/strax/issues/307
cd9fd4c Double check Nones are not returned by CMT https://github.com/XENONnT/straxen/issues/255
2492e23 fix documentation on cut-plugins https://github.com/XENONnT/straxen/issues/247 

**Can you briefly describe how it works?**
Fix some open issues that were still lingering.

### CODE REVIEW

**Comment by jorana** [comment]
@ahiguera-mx , do you agree with the additional check in cd9fd4c?

**Comment by ahiguera-mx** [comment]
> @ahiguera-mx , do you agree with the additional check in [cd9fd4c](https://github.com/XENONnT/straxen/commit/cd9fd4c4d7b09346eee6812862812a2705e09a36)?

Hi @jorana, yes thanks for catching this, although that should never happened if the gain model is correct, but better to be safe than sorry

**Review by ahiguera-mx — APPROVED** [review]

File: straxen/corrections_services.py [hunk 1]
```diff
@@ -167,7 +171,7 @@ def get_pmt_gains(self, run_id, model_type, global_version):
                                  f'{type(global_version)}')
 
             # Generate an array of values and multiply by the 'global_version'
-            to_pe = np.ones(n_tpc_pmts, dtype=np.float32) * global_version
+            to_pe = np.ones(n_tpc_pmts, dtype=gain_dtype) * global_version
```
Comment (WenzDaniel) [inline straxen/corrections_services.py:174]: Why can the dtype of the gains change?

**Review by WenzDaniel — APPROVED** [review]
Looks fine just one question to be answered

File: straxen/corrections_services.py (hunk 1 above)
Comment (jorana) [reply straxen/corrections_services.py:174]: Not per se something that should be changed, therefore I made it a kwarg. Could of course also have it hardcoded in the function. I like it more than saying `np.ones(n_tpc_pmts, dtype=np.float32)` twice (also some lines down), I rather define it once.

**Pull request merged**